pages/linux/veracrypt.md

```diff
@@ -0,0 +1,12 @@
+# veracrypt
+
+> Free and open source disk encryption software.
+> More information: <https://www.veracrypt.fr/code/VeraCrypt/plain/doc/html/Documentation.html>.
+
+- Decrypt a volume using a password and mount it to a directory:
+
+`veracrypt --password={{password}} {{path/to/volume}} {{path/to/mount_point}}`
```

Comment:
Is the `path/to/volume` a partition like `/dev/sda3`? Then, I suggest:
```suggestion
`veracrypt --password={{password}} {{/dev/sdXN}} {{path/to/mount_point}}`
```

Reply:
It can be either a partition or a file. In the example above I was thinking as a file and didn't consider as a partition.

Reply:
Please add it.

Reply:
048117906d7e330e817039df8a40e4710de3f275